modules/monitoring-disabling-the-local-alertmanager.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+[id="disabling_the_local_alertmanager_{context}"]
+= Disabling the local Alertmanager
+
+A default installation of the {product-title} monitoring stack includes in the `openshift-monitoring` project an Alertmanager instance that routes alerts from Prometheus instances.
+
+You can disable this Alertmanager instance by configuring the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project.
+You might do so in single-node deployments that do not need to deploy a local Alertmanager cluster.
```

Comment:
since we can disable Alertmanager in multi-node cluster. change to below is better
`You might do so if you do not need the local Alertmanager cluster.`

Reply:
Ok. That's simpler and more accurate, then.

Reply:
Combining the two sentences makes it more direct: 

"If you do not need the local Alertmanager instance, you can disable it by configuring the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project."